src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -551,6 +635,63 @@ void PeHeuristics::getSecuROMHeuristics()
 	}
 }
 
+/**
+ * Detection of MPRMMGVA packer
+ */
+void PeHeuristics::getMPRMMGVAHeuristics()
+{
+	const uint8_t * fileData = reinterpret_cast<const uint8_t *>(search.getPlainString().c_str());
+	const uint8_t * filePtr = fileData + toolInfo.epOffset;
+	const uint8_t * fileEnd = fileData + search.getPlainString().length();
+	unsigned long long offset1;
+
+	// Skip up to 8 NOPs or JMPs
+	filePtr = skip_NOP_JMP8_JMP32(fileData, filePtr, fileEnd, 8);
+	if (filePtr != nullptr)
```

Comment:
Nitpick: Consider unifying the style between `if (cond)` and `if(cond)`. Both forms appear in this PR. Generally, we strive to use the same format, at least on the level of a single file.